kiva/examples/kiva/image_explorer.py

```diff
@@ -37,7 +37,7 @@ def _draw_mainlayer(self, gc, view_bounds=None, mode="default"):
         if self.image is None:
             return
 
-        gc.draw_image(self.image, (0, 0, gc.width(), gc.height()))
+        gc.draw_image(self.image, (0, 0, self.width, self.height))
```

Comment:
This got broken by HiDPI support. The correct approach is to use the component bounds.